string type
geometry_msgs/Pose pose
geometry_msgs/Polygon geometry
